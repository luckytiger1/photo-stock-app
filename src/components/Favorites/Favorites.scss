.favorites-title {
  margin: 50px 0;
  h2 {
    font-family: 'SF UI Display Light', Helvetica, sans-serif;
    font-weight: 900;
    font-size: 45px;
    letter-spacing: 2px;
  }
}
