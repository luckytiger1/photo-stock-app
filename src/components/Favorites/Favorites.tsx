import React from 'react';
import './Favorites.scss';
import GalleryContainer from '../Gallery/GalleryContainer';

const Favorites: React.FC = () => {
  return (
    <div className="favorites-container">
      <div className="favorites-title">
        <h2 className="text-center">Favorites</h2>
      </div>
      <GalleryContainer />
    </div>
  );
};

export default React.memo(Favorites);
