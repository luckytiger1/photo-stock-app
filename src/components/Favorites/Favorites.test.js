import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import Favorites from './Favorites';

it('<Favorites /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<Favorites />)).toMatchSnapshot();
});
