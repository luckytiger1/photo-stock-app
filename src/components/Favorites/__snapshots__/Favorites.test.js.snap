// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Favorites /> renders correctly 1`] = `
<div
  className="favorites-container"
>
  <div
    className="favorites-title"
  >
    <h2
      className="text-center"
    >
      Favorites
    </h2>
  </div>
  <withRouter(Connect(GalleryContainer)) />
</div>
`;
